backend/document/domain/assembly_strategies.py

```diff
@@ -5060,10 +5055,6 @@ def assemble_tw_as_iterator_for_book_then_lang(
     # Sort resources by language
     key = lambda resource: resource.lang_code
     # FIXME Do we need to sort anything other than
```

Comment:
Looks like this comment got cut in half -- I recommend restoring it or deleting if no longer needed.